### PR TITLE
React Element accepted as navbar logo

### PR DESCRIPTION
**Pull request by mveres**
example: 
image

### CODE REVIEW

File: mdx-layout/src/NavBar.tsx [hunk 1]
```diff
@@ -25,22 +25,26 @@ export const NavBar = ({
...
-      defaultChecked={true}
-      style={{ display: 'none' }}
-    />
-    <NavGroup items={graph.overallOrder(true)} graph={graph} />
-  </nav>
-);
+  logo?: string | ReactElement;
+}) => {
+  const Logo = () =>
+    typeof logo === 'string' ? <img src={logo} /> : <>{logo}</>;
```
Comment (gqio) [inline mdx-layout/src/NavBar.tsx:31]: can't we remove the fragment here `<>` ?

**Comment by georges-gomes** [comment]
Remove the "name" everything can and should be done in logo.

Like greg: can we remove the fragment?

File: mdx-layout/src/NavBar.tsx (hunk 1 above)
Comment (mveres) [reply mdx-layout/src/NavBar.tsx:31]: yes

File: mdx-layout/src/NavBar.tsx [hunk 2]
```diff
@@ -20,27 +20,34 @@ const ThreeBars = () => (
...
-    <NavGroup items={graph.overallOrder(true)} graph={graph} />
-  </nav>
-);
+  logo?: string | ReactElement;
+}) => {
+  const Logo = () => (typeof logo === 'string' ? <img src={logo} /> : logo);
+  return (
+    <nav>
+      <h3
+        style={{
```
Comment (gqio) [inline mdx-layout/src/NavBar.tsx:32]: I won't go this but rather in the NavBar.scsss

**Pull request merged**